entities/reader.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+require_relative 'error'
+
+class Reader # :nodoc:
+  attr_reader :name, :email, :city, :street, :house
+  def initialize(name, email, city, street, house)
```

Comment:
It's not ok that you have 5 args in your initializer.

Try to use `named params` or `hash params`.